atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/TimestampChecker.java

```diff
@@ -15,17 +15,28 @@
  */
 package com.palantir.atlasdb.jepsen;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
 
 import com.google.common.collect.ImmutableMap;
 import com.palantir.atlasdb.jepsen.events.Event;
+import com.palantir.atlasdb.jepsen.events.EventVisitor;
 
 import clojure.lang.Keyword;
 
 public final class TimestampChecker {
```

Comment:
Doesn't need to be `final` anymore. Should also probably be called something else like `JepsenCheckerConverter` (or a much better name that I can't think of atm :) )